nuggetsapp/models.py

```diff
@@ -1,6 +1,9 @@
 from django.db import models
 from django.utils import timezone
 from django.contrib.auth.models import User
+import datetime
+from datetime import timedelta
```

Comment:
Best practice here is to do only one of the following:
1) only `import datetime`, and always reference things in the datetime module in code.  Eg. `datetime.timedelta(days=1)` or `datetime.date.today()`
2) only import the children properties of datetime that you care about, and reference them directly.  Eg. `from datetime import timedelta, date` so you can later do `timedelta(days=1)` or `date.today()`

Reason is, the way that it's written now, accessing children properties of datetime can get confusing in certain contexts.  Better to keep things consistent.